crates/burn-import/onnx-tests/build.rs

```diff
@@ -32,7 +32,8 @@ fn main() {
         .input("tests/exp/exp.onnx")
         .input("tests/expand/expand.onnx")
         .input("tests/flatten/flatten.onnx")
-        .input("tests/gather/gather.onnx")
```

Comment:
If we're not using this file anymore due to the rename, we should remove it from the workspace 🙂